README.rst

```diff
@@ -190,10 +190,13 @@ For example, using Gunicorn:
     $ pip install httpbin
     $ gunicorn httpbin:app
 
-Or run it directly:
+Or install and run it directly:
 
 .. code:: bash
 
+    $ git clone git@github.com:Runscope/httpbin.git
+    $ cd httpbin
+    $ [sudo] python setup.py install
```

Comment:
@wdschei for several reasons:

1. You should be using a virtualenv for local development/usage
2. You should be using `pip` to install it, e.g., `pip install -e .`
3. People who tend to read this documentation, are not deploying HTTPbin but running it locally and installing the package globally is a poor choice for them that will only cause numerous headaches in the future.